packaging/linux/run_keybase

```diff
@@ -42,12 +45,13 @@ logdir="${XDG_CACHE_HOME:-$HOME/.cache}/keybase"
 mkdir -p "$logdir"
 
 echo Launching keybase service...
-keybase service &>> "$logdir/keybase.service.log" &
+keybase -d --log-file="$logdir/keybase.service.log" service &>> "$logdir/keybase.start.log" &
```

Comment:
I wonder if the logging-to-two-different-files approach will work well when/if we switch to a systemd way of doing things. How does it work with launchd on OSX?